js/src/repo.js

```diff
@@ -46,26 +56,36 @@ module.exports = (common) => {
     })
 
     it('.stat', (done) => {
-      ipfs.repo.stat((err, stat) => {
+      ipfs.repo.stat((err, res) => {
         expect(err).to.not.exist()
-        expect(stat).to.exist()
-        expect(stat).to.have.property('numObjects')
-        expect(stat).to.have.property('repoSize')
-        expect(stat).to.have.property('repoPath')
-        expect(stat).to.have.property('version')
-        expect(stat).to.have.property('storageMax')
+        expect(res).to.exist()
```

Comment:
These assertions are repeated all over the tests. We should extract the assertions into a function and pass in `res` instead.

```
const expectIsStat = (stat) => {
  // Make all the assertions once here
}
```